ci/jenkins/test-rancher.sh

```diff
@@ -0,0 +1,336 @@
+    export GOROOT=/usr/local/go
+    export GOCACHE="${WORKSPACE}/../gocache"
+    export PATH=${GOROOT}/bin:$PATH
+
+    git show --numstat
+    make clean
+    ${CLEAN_STALE_IMAGES}
+    if [[ ! "${TESTCASE}" =~ "e2e" && "${DOCKER_REGISTRY}" != "" ]]; then
+        docker pull "${DOCKER_REGISTRY}/antrea/sonobuoy-systemd-logs:v0.3"
+        docker tag "${DOCKER_REGISTRY}/antrea/sonobuoy-systemd-logs:v0.3" "sonobuoy/systemd-logs:v0.3"
+    fi
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
+    DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
+    make flow-aggregator-image
+
+    # Enable verbose log for troubleshooting.
+    sed -i "s/--v=0/--v=4/g" build/yamls/antrea.yml
+
+    # echo "=== Fill serviceCIDRv6 and serviceCIDR ==="
+    # # It is unnecessary for cluster with AntreaProxy enabled.
+    # SVCCIDRS=$(kubectl cluster-info dump | grep service-cluster-ip-range | head -n 1 | cut -d'=' -f2 | cut -d'"' -f1)
+    # echo "Service CIDRs are $SVCCIDRS"
+    # regexV6="^([0-9a-fA-F]{0,4}:){1,7}[0-9a-fA-F]{0,4}.*$"
+    # IFS=',' read -ra CIDRS <<< "$SVCCIDRS"
+    # for cidr in "${CIDRS[@]}"; do
+    #     if [[ ${cidr} =~ ${regexV6} ]]; then
+    #         sed -i "s|#serviceCIDRv6:|serviceCIDRv6: ${cidr}|g" build/yamls/antrea.yml
+    #     else
+    #         sed -i "s|#serviceCIDR: 10.96.0.0/12|serviceCIDR: ${cidr}|g" build/yamls/antrea.yml
+    #     fi
+    # done
+
+    echo  "=== Append antrea-prometheus.yml to antrea.yml ==="
+    echo "---" >> build/yamls/antrea.yml
+    cat build/yamls/antrea-prometheus.yml >> build/yamls/antrea.yml
+    control_plane_ip="$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 ~ role {print $6}')"
+    # rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" build/yamls/*.yml jenkins@[${control_plane_ip}]:${WORKDIR}/
+    echo ${control_plane_ip}
+    cp -f build/yamls/*.yml $WORKDIR
+    #scp -q -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i "${WORKDIR}/.ssh/id_rsa" build/yamls/*.yml ubuntu@[${control_plane_ip}]:~/
+    #rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" build/yamls/*.yml jenkins@[${control_plane_ip}]:~/
+
+    echo "====== Delivering Antrea to all the Nodes ======"
+    docker save -o antrea-ubuntu.tar antrea/antrea-ubuntu:latest
+    docker save -o flow-aggregator.tar antrea/flow-aggregator:latest
+    kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 !~ role {print $6}' | while read IP; do
+        rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" antrea-ubuntu.tar ubuntu@[${IP}]:~/antrea-ubuntu.tar
+        rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no" flow-aggregator.tar ubuntu@[${IP}]:~/flow-aggregator.tar
+        ssh -o StrictHostKeyChecking=no -n ubuntu@${IP} "${CLEAN_STALE_IMAGES}; docker load -i ~/antrea-ubuntu.tar; docker load -i ~/flow-aggregator.tar" || true
```

Comment:
If the runtime of kubernetes node is containerd, you should use "crictl" or "ctr" command to import containerd image.

Reply:
runtime is docker